src/inspection.py

```diff
@@ -0,0 +1,81 @@
+import json
+from collections import defaultdict
+
+import geopandas as gpd
```

Comment:
Hi @jstaszel-agh, you are right the conda lock .yml config did not include some packages such as geopandas. Thank you for pointing this out. I seem to have resolved the issue, I created a fresh environment from the lock file and ran the code and the project seems to work fine now. I advise you to delete the previously created env and create it with conda-lock again or just create it again under a different name. Please let me know in case there's additional debuggining needed. Thank you!